Update README.md (#253)
autovance/ftp-srv: README.md

__Default:__ `"ftp://127.0.0.1:21"`

#### `pasv_url`

> This must be the external IP address where the FTP Server service is bind as `ftp-srv` provides this IP address as part of the response to the FTP Client when a `PASV` command is received in the attempting of a passive connection. Reference [PASV verb](https://cr.yp.to/ftp/retr.html#pasv).

Reviewer: ```suggestion
This **must** be the external WAN **IP address** that the FTP server is bound to. `FTP-srv` provides this IP address to the client when a `PASV` command is received in the handshake for a passive connection. Reference [PASV verb](https://cr.yp.to/ftp/retr.html#pasv).
```
